### PR TITLE
Add newspost for playtest-20150405

### PR DESCRIPTION
**Pull request by obrakmann**

### CODE REVIEW

**Comment by Mailaender** [comment]
:+1: although it lacks the usual Tiberian Sun screenshot clickbait. :smile:

**Comment by pchote** [comment]
Looks good to me.

**Comment by obrakmann** [comment]
yes, I know it's lacking pictures, but there's nothing relevant to show off. Unless someone has some real nice eye candy, I wouldn't know what images to add.

File: content/news/playtest-20150405.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+We have received lots of feedback on our last playtest, for which we are very grateful. Let me take this opportunity to thank all our testers!
+
+In fact, we have received so much feedback that we felt it was worth it to do another, hopefully last, playtest before the release.
+
+This new playtest addresses most of the issues that have been reported. In particular, these are:
+
+  - The balancing in the Red Alert mod has received some more touch-ups.
```
Comment (ScottNZ) [inline content/news/playtest-20150405.md]: While this is certainly true, I still have a bit of work to do on this and some people might think that the balance has been finalised with this playtest :s

**Comment by Mailaender** [comment]
You could use the screenshots from https://github.com/OpenRA/OpenRA/pull/7722#issuecomment-89305122. User interface screenshots are better than no screenshots I guess. Also the problem is easy to understand for non-technical gamers.

**Comment by Mailaender** [comment]
Maybe also highlight the new blend modes for Dune 2000 from https://github.com/OpenRA/OpenRA/pull/7811.

image

image

**Comment by Mailaender** [comment]
I tagged https://github.com/OpenRA/OpenRA/releases/tag/playtest-20150405 so you can update the downloads site.

**Comment by penev92** [comment]
:+1:

**Comment by obrakmann** [comment]
wait another moment with merging, need to address Scott's comment

**Comment by Mailaender** [comment]
https://build.opensuse.org/package/rdiff/games:openra/playtest?linkrev=base&rev=12

**Pull request merged**